src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import PublicMessage from './components/PublicMessage.jsx';
+import PublicMessageBoard from './components/PublicMessageBoard.jsx';
+import useConfig from './hooks/useConfig.js';
+import usePublicMessages from './hooks/usePublicMessages.js';
+
+/**
+ * Application entrypoint.
+ *
+ * @constructor
+ * @return {JSX.Element}
+ */
+export default function App() {
+  const {infoPoint, routes} = useConfig();
+  const publicMessages = usePublicMessages(infoPoint, routes);
+
+  return (
+    <PublicMessageBoard>
+      {(publicMessages === undefined) ? null : (publicMessages === null) ? (
```

Comment:
Looking at this again it's probably not communicated very well but `undefined` will result in a blank page due to the `null` interpolation.

I went with that since most of the time, a "Loading" indicator will flash for like a millisecond, would you prefer to have some sort of indicator?

(that being said I'll at least update the code to communicate what's going on a little better)

Reply:
Ah, nope I just got lost in the `? : (...)`s.